ansible/roles/cluster_infra/templates/resources.tf.j2

```diff
@@ -224,7 +257,29 @@ resource "openstack_networking_port_v2" "control" {
   }
 }
 
+# Storage VLAN
+
+resource "openstack_networking_port_v2" "control_portal_storage" {
+  name           = "{{ cluster_name }}-control-0-portal-storage"
+  network_id     = data.openstack_networking_network_v2.portal_storage_direct.id
+  admin_state_up = "true"
+
+  # not sure if needed here
+  security_group_ids = [
+    "${openstack_networking_secgroup_v2.secgroup_slurm_cluster.id}"
+  ]
+
+  # for now, until we set up flavors
+  binding {
+    vnic_type = "normal"
```

Comment:
lets add cluster_vnic_type a bit like the existing one:

`vnic_type = "{{ cluster_vnic_type | default('normal') }}"`

Reply:
Same for other port. And in fact it'd be nice to make it clear that this doesn't apply to the non-storage ports, so something like `cluster_storage_vnic_type`.